lib/webdriveragent.js

```diff
@@ -121,12 +113,14 @@ class WebDriverAgent {
     try {
       let carthagePath = await fs.which('carthage');
       log.debug(`Carthage found: ${carthagePath}`);
-    } catch (err) {
-      log.info('Carthage not found. Install using `brew install carthage`');
-    }
-    if (!await fs.hasAccess(`${this.bootstrapPath}/Carthage`)) {
-      log.debug('Running WebDriverAgent bootstrap script to install dependencies');
+      if (!await fs.hasAccess(`${this.bootstrapPath}/Carthage`)) {
+        log.debug('Running WebDriverAgent bootstrap script to install dependencies');
+      } else {
+        log.debug('Running WebDriverAgent bootstrap script to update dependencies');
```

Comment:
@imurchie is this only a problem when people are running appium from source and receive wda updates?